py3status/modules/wifi.py

```diff
@@ -66,16 +68,34 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = DEFAULT_FORMAT
     round_bitrate = True
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
 
     def post_config_hook(self):
+        # DEPRECATION WARNING
+        format_down = getattr(self, 'format_down', None)
+        format_up = getattr(self, 'format_up', None)
+        format = getattr(self, 'format', None)
```

Comment:
I think it's not working great right now. I also get this WARNING.
```
2017-04-09 04:15:42 WARNING Instance `wifi`, user method `wifi` failed (AttributeError) wifi.py line 175.
2017-04-09 04:15:42 INFO Traceback
AttributeError: Py3status instance has no attribute '_ssid'
  File "/usr/lib/python2.7/site-packages/py3status-3.6rc1-py2.7.egg/py3status/module.py", line 709, in run
    response = method()
  File "/home/chris/src/py3status/py3status/modules/wifi.py", line 175, in wifi
    if self._ssid != ssid:
```
We probably could fix up `wifi` better at some other time. I think the annoying part about this is it sucks at detecting devices. It just gives us `wlan0`. I'm on something like `wlp3s0`. Using netbook for this one...

Will rename `status` back to `full_text`. 🚶🕺